lib/api/bucketDeleteTagging.js

```diff
@@ -0,0 +1,50 @@
+const collectCorsHeaders = require('../utilities/collectCorsHeaders');
+const { metadataValidateBucket } = require('../metadata/metadataUtils');
+const { pushMetric } = require('../utapi/utilities');
+const metadata = require('../metadata/wrapper');
+const util = require('node:util');
+
+/**
+ * Bucket Delete Tagging - Delete a bucket's Tagging
+ * @param {AuthInfo} authInfo - Instance of AuthInfo class with requester's info
+ * @param {object} request - http request object
+ * @param {object} log - Werelogs logger
+ * @param {function} callback - callback to server
+ * @return {undefined}
+ */
+async function bucketDeleteTagging(authInfo, request, log, callback) {
+    log.debug('processing request', { method: 'bucketDeleteTagging' });
+
+    let bucket;
+    const bucketName = request.bucketName;
+    const metadataValParams = {
+        authInfo,
+        bucketName,
+        requestType: 'bucketDeleteTagging',
+    };
+
+    try {
+        let promisifyFunc = util.promisify(metadataValidateBucket);
+        bucket = await promisifyFunc(metadataValParams, log);
```

Comment:
Maybe you already discussed that with @miniscruff but I am not a big fan of try/catch to catch Promise errors...
Why not trying to stick with golang-like logic using https://www.npmjs.com/package/await-to-js ?

Reply:
Otherwise LGTM

Reply:
Hmm, not sure the issue with try/catch but for JS it is definitely the standard and most common approach. I think we might want to make our try catches smaller, as in one around each thing that can error. But I would not be a fan of using that package, don't get me wrong, I really like go syntax but for JS it will put us on an island as it is not standard at all.